### PR TITLE
Improved comment indentation

### PR DESCRIPTION
**Pull request by meets2tushar**
As a [GCI task] (https://codein.withgoogle.com/dashboard/task-instances/6400773606866944/)
improved comment section.

### CODE REVIEW

File: src/lib_ccx/asf_constants.h [hunk 1]
```diff
@@ -1,94 +1,94 @@
...
 	int VideoStreamNumber;
 	int AudioStreamNumber;
 	int CaptionStreamNumber;
-	int CaptionStreamStyle; // 1 = NTSC, 2 = ATSC
-	int DecodeStreamNumber; // The stream that is chosen to be decoded
-	int DecodeStreamPTS; // This will be used for the next returned block
-	int currDecodeStreamPTS; // Time of the data returned by the function
-	int prevDecodeStreamPTS; // Previous time
-	int VideoStreamMS; // See ableve, just for video
+	int CaptionStreamStyle;      // 1 = NTSC, 2 = ATSC
```
Comment (canihavesomecoffee) [inline src/lib_ccx/asf_constants.h]: This alignment is good.

File: src/lib_ccx/asf_constants.h [hunk 2]
```diff
@@ -1,94 +1,94 @@
...
 #define ASF_BINARY_MEDIA "\xE2\x65\xFB\x3A\xEF\x47\xF2\x40\xAC\x2C\x70\xA9\x0D\x71\xD3\x43"
 
-// ASF_BINARY_MEDIA : Major media types
+                                  // ASF_BINARY_MEDIA : Major media types
 #define DVRMS_AUDIO "\x9D\x8C\x17\x31\xE1\x03\x28\x45\xB5\x82\x3D\xF9\xDB\x22\xF5\x03"
 #define DVRMS_NTSC "\x80\xEA\x0A\x67\x82\x3A\xD0\x11\xB7\x9B\x00\xAA\x00\x37\x67\xA7"
 #define DVRMS_ATSC "\x89\x8A\x8B\xB8\x49\xB0\x80\x4C\xAD\xCF\x58\x98\x98\x5E\x22\xC1"
 
-// 10.13 - Undocumented DVR-MS properties
+                                 // 10.13 - Undocumented DVR-MS properties
```
Comment (canihavesomecoffee) [inline src/lib_ccx/asf_constants.h]: Why is this aligned this way (same with the ones above?)

File: src/lib_ccx/avc_functions.h [hunk 3]
```diff
@@ -5,10 +5,10 @@
 struct avc_ctx
 {
 	unsigned char cc_count;
-	// buffer to hold cc data
+	                          // buffer to hold cc data
```
Comment (canihavesomecoffee) [inline src/lib_ccx/avc_functions.h]: This does not add much to the readability

Comment (meets2tushar) [inline src/lib_ccx/avc_functions.h]: That was an small code;  then i prefer to go for alignment of all the cooments

**Comment by cfsmp3** [comment]
@meets2tushar Can you submit a new PR with only the changes @canihavesomecoffee says are OK? It's a lot easier if we can merge a few things first and then deal with the rest. If there's changes that can be merged already let's take care of them already.

**Comment by meets2tushar** [comment]
@cfsmp3  I am not able to make another (a new ) PR. It is showing me a page of comparison only.

**Comment by cfsmp3** [comment]
@meets2tushar I don't understand, sorry. You can just close this PR and start a new one with the files Willem said.

**Pull request closed without merging**